backend/matechai/src/main/java/com/ft/matechai/auth/node/VerificationToken.java

```diff
@@ -17,5 +17,5 @@ public class VerificationToken {
     @Id
     private String token;
 
-    private Long userId;
+    private String userId;
```

Comment:
```suggestion
    private String username;
```
Please  change field name !